SubmittedAssignments/Arti_Kadam/preview.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css">
+    <title>visiting card</title>
+</head>
+<body>
+    <center style="padding-top:40px">
+        <div class="card1-wrapper card-front">
```

Comment:
**card1-wrapper card-front** not valid
**card-wrapper card-front** should be like this